_posts/2020-09-03-tutorial-syllabics-fonts.md

```diff
@@ -0,0 +1,283 @@
+a font that displays syllabics correctly.
+
+## Common syllabics fonts
+
+I am aware of the following fonts:
+
+ - **Aboriginal Sans** and **Aboriginal Serif**
+ - **Euphemia**
+ - **Gadugi**
+ - **Noto Sans Canadian Aboriginal**
+
+I made an **interactive demonstration** of these fonts that you can play
+with. Go ahead and type any text in (western) Cree into the box and it
+will be converted to syllabics and displayed in the appropriate font, if
+it's available on your system:
+
+<p class="codepen" data-height="500" data-theme-id="light" data-default-tab="result" data-user="eddieantonio" data-slug-hash="GRZOBYY" style="height: 500px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;" data-pen-title="Syllabics font playground">
+  <span>See the Pen <a href="https://codepen.io/eddieantonio/pen/GRZOBYY">
+  Syllabics font playground</a> by Eddie Antonio Santos (<a href="https://codepen.io/eddieantonio">@eddieantonio</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+
+---
+
+But **where** do these fonts come from? Where should you **expect these fonts to be
+available**?
+
+
+### Aboriginal Sans and Aboriginal Serif
+
+As the names suggest, they have both serif and sans-serif variants for
+writing in Latin script (e.g., English, French, etc.).
+
+#### Who created these fonts?
+
+These fonts were created by the Language Geek himself, [Chris Harvey][].
+
+#### Where can I find these fonts?
+
+As far as I know, no system has Aboriginal Sans or Aboriginal Serif
+pre-installed, so you have to [download them yourself from his
+website][languagegeek-fonts]. Sometimes, they are bundled with certain
+[Keyman][] keyboard layouts.
+
+If you've tried getting syllabics to work on your computer, chances are,
+you've downloaded either of these fonts!
+
+[Languagegeek.com][languagegeek-fonts] has many other fonts that
+display syllabics.
+
+[Chris Harvey]: http://www.languagegeek.com/
+[languagegeek-fonts]: http://www.languagegeek.com/font/fontdownload.html
+[Keyman]: https://keyman.com/
+
+
+### Gadugi
+
+This is a modern font that displays both syllabics and Cherokee.
+“Gadugi” is a Cherokee word that means “[cooperative
+labour][gadugi-def]”.
+
+#### Who created this font?
+
+I believe Microsoft commissioned this font.
+
+#### Where can I find this font?
+
+Gadugi has been included in [**all versions of Windows since Windows
+8**][gadugi-availability]. [It's also available as a “Cloud
+font”][gadugi-office] on Microsoft Office products.
+
+[gadugi-def]: https://archive.org/details/cherokeeenglishd0000feel/page/92/mode/2up?q=gadugi
+[gadugi-availability]: https://docs.microsoft.com/en-us/typography/font-list/gadugi#products-that-supply-this-font
+[gadugi-office]: https://support.microsoft.com/en-us/office/cloud-fonts-in-office-f7b009fe-037f-45ed-a556-b5fe6ede6adb?ui=en-us&rs=en-us&ad=us#text:~:text=gadugi.ttf
+
+
+### Euphemia
```

Comment:
As discussed in the meeting today, it may be worth discussing the syllabic orientation issues as a note in this section (and possibly move the entire discussion of Euphemia down in the document).